src/Layout/XmlLayoutSchemaLog4j.cs

```diff
@@ -28,7 +28,7 @@
 namespace log4net.Layout
 {
 	/// <summary>
-	/// Layout that formats the log events as XML elements compatible with the log4j schema
+	/// Layout that formats the log events as XML elements compatible with the log4j 1.2 schema
```

Comment:
The newly added layout class claims to do this, too. Would you please explain this change?

Reply:
the class has a version field for historical reasons but only accepts 1.2 as value. To me it is more important to say it is the 1.x version and not the 2.x version of log4j.